### ISSUE TITLE
1.13 fails to build

### ISSUE THREAD

**Issue by nbecker** [bashtage/ng-numpy-randomstate#92]
fedora linux 25 (gcc-6.3.1)

    ./randomstate/pcg64.c:3033:14: error: request for member ‘high’ in something not a structure or union
       __pyx_v_out.high = __pyx_t_2;
                  ^
...
    ./randomstate/pcg64.c:3046:14: error: request for member ‘low’ in something not a structure or union
       __pyx_v_out.low = __pyx_t_2;

**Comment by bashtage** [comment]
Hmmm, not really sure what is going on.  Is this possibly a GCC bug, or something set by a flag?

Which cython are you using?

**Comment by nbecker** [comment]
python3-Cython-0.25.2-3.fc25.x86_64

BTW, do you know if I can use pip to re-install an older version (1.12)?  pip seems to have left me with a broken randomstate (even though the failed install should have done a rollback) :(

**Comment by bashtage** [comment]
pip install randomstate=1.12 should work

**Comment by nbecker** [comment]
1.12 was reinstalled, but I get (on my app):
AttributeError: 'randomstate.xorshift1024.RandomState' object has no
attribute 'random_uintegers'
Wasn't random_uintegers available in 1.12?

**Comment by bashtage** [comment]
I put it back in right after 1.12.

What happens if you install from github instead of pip?

git clone https://github.com/bashtage/ng-numpy-randomstate
cd ng-numpy-randomstate <https://github.com/bashtage/ng-numpy-randomstate>
python setup.py install

Kevin

**Comment by nbecker** [comment]
compiled without error (but there are a few warnings you might want to look
at sometime)

**Comment by nbecker** [comment]
But now I'm back to (with randomstate==1.13+2.gea9accf)
AttributeError: 'randomstate.xorshift1024.RandomState' object has no
attribute 'random_uintegers'

**Comment by bashtage** [comment]
Hmmm...

**Comment by nbecker** [comment]
OK, I cleaned out all the old versions of randomstate in site_packages, and reinstalled the current git, and everything is OK.
As to why it didn't build from pip, that remains a problem.

**Comment by bashtage** [comment]
Run 

```
sudo pip uninstall randomstate -y
sudo pip uninstall randomstate -y
```
(yes twice) then reinstall. (Obviously only sudo if needed)

**Comment by bashtage** [comment]
Something is wrong with the pip sdist -- it fails on Ubuntu/Mint as well.

**Comment by bashtage** [comment]
Thanks for reporting -- I think I need to not publish the c files in the sdist since I don't think these are compatible across Python 2.7 and 3.x.  Not 100% but this is my suspicion.  I published from Windows/2.7 so this might be why you had an issue.  I'll probably clean things up and publish a 1.13.2 soon that requires Cython to do a sdist install.

**Comment by bashtage** [comment]
This is not fixed -- have to use local Cython directly since there is some platform conditional Cython being used.
